pkg/apis/networkaddonsoperator/v1/networkaddonsconfig_types.go

```diff
@@ -2,30 +2,47 @@ package v1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared"
+)
 
 // NetworkAddonsConfigSpec defines the desired state of NetworkAddonsConfig
-type NetworkAddonsConfigSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
-}
+// +k8s:openapi-gen=true
```

Comment:
So if we want to add new types in v1, should those be added here as well?

Reply:
yes